content/docs/2.1/scalers/mongo.md

```diff
@@ -0,0 +1,80 @@
+# mongo with ScaledJob
+
+## Trigger Specification
+This specification describes the mongo trigger that scales based on result of Mongo query.
+
```

Comment:
Can you add a summary similar to https://github.com/kedacore/keda-docs/blob/master/content/docs/2.1/scalers/azure-service-bus.md#L14 here please?